test/pages/index.test.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import Home from '../../pages/index';
+import { getProjects } from '../../util';
 import { render } from '../testUtils';
 
 describe('Home page', () => {
-  it('matches snapshot', () => {
-    const { asFragment } = render(<Home numRepos={0} recentEvents={[]} />, {});
+  it('matches snapshot', async () => {
+    const projects = await getProjects();
```

Comment:
I would not rely on `getProjects` to work for this snapshot test; instead, we should pass in a static array of known values, i.e. a fixture. What happens, for example, if we run this test without internet? Or if the GitHub API changes?

Separately, it would be a good idea to test `getProjects`! Just want to separate the concerns of the tests here.